src/Compilers/VisualBasic/Test/Symbol/SymbolsTests/Source/EventTests.vb

```diff
@@ -24,6 +24,32 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.UnitTests
     Public Class EventSymbolTests
         Inherits BasicTestBase
 
+        <WorkItem(20335, "https://github.com/dotnet/roslyn/issues/20335")>
+        <Fact()>
+        Public Sub ProtectedHandler()
+            Dim source = <compilation name="F">
+                             <file name="F.vb">
+ Public Class Form1
+    Protected Event EnterOriginal As System.Action
```

Comment:
Do we have a test that `MyBase` works for other accessibility modifiers: Friend for example? If not we should add them here.